bentoml/_internal/io_descriptors/pandas.py

```diff
@@ -686,7 +730,7 @@ def infer(input_series: pd.Series) -> pd.Series:
     def __init__(
         self,
         orient: ext.SeriesOrient = "records",
-        dtype: ext.PdDTypeArg | None = None,
+        dtype: ext.PdDTypeArg | str | None = None,
```

Comment:
I think that `str` should be covered by `PdDTypeArg`: it contains `Dtype`, `Dtype` contains `NpDtype`, and that contains `str`.

```suggestion
        dtype: ext.PdDTypeArg | None = None,
```

Reply:
Thanks for pointing this out!  `str` is covered by PdDTypeArg.  I have removed it.

Reply:
seems like you haven't updated this yet?